src/XrdOuc/XrdUtils.hh

```diff
@@ -0,0 +1,59 @@
+//------------------------------------------------------------------------------
+// Copyright (c) 2024-2025 by European Organization for Nuclear Research (CERN)
+// Author: Cedric Caffy <cedric.caffy@cern.ch>
+//------------------------------------------------------------------------------
+// XRootD is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// XRootD is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with XRootD.  If not, see <http://www.gnu.org/licenses/>.
+//------------------------------------------------------------------------------
+
+
+/**
+ * PRIVATE HEADER for utility functions, implementation in XrdOucUtils.cc
+ */
+#ifndef XROOTD_XRDUTILS_HH
+#define XROOTD_XRDUTILS_HH
```

Comment:
@abh3 Please let us know your opinion on the naming here. I'd suggest `XrdOucPrivUtils.hh`, and we only install into `include/xrootd/private/XrdOuc` if asked by someone to keep it truly private for as long as possible.

Reply:
I've renamed it to `XrdOucPrivateUtils.hh`, then we know what is this header about